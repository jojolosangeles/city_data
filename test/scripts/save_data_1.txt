load data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv as sample20
columns sample20
drop sample20 DR Number, Date Reported, Area ID, Reporting District, Crime Code, Crime Code Description
drop sample20 MO Codes, Premise Code, Premise Description, Location
columns sample20
save sample20
