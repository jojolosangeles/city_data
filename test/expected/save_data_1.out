Test: scripts/save_data_1.txt
> load data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv as sample20
Loaded data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv into DataFrame sample20
> columns sample20
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
> drop sample20 DR Number, Date Reported, Area ID, Reporting District, Crime Code, Crime Code Description
Dropping ['DR Number', 'Date Reported', 'Area ID', 'Reporting District', 'Crime Code', 'Crime Code Description'] from DataFrame 'sample20'
Columns dropped
> drop sample20 MO Codes, Premise Code, Premise Description, Location
Dropping ['MO Codes', 'Premise Code', 'Premise Description', 'Location'] from DataFrame 'sample20'
Columns dropped
> columns sample20
['Date Occurred' 'Time Occurred' 'Area Name' 'Victim Age' 'Victim Sex'
 'Victim Descent' 'Address' 'Cross Street']
> save sample20
Saved CSV File 'sample20.csv'
Test completed: scripts/save_data_1.txt
